Makefile

```diff
@@ -1,8 +1,32 @@
 SNAPPY = $(CHAIN)/vendor/github.com/google/snappy
 ROCKSDB = $(CHAIN)/vendor/github.com/facebook/rocksdb
+DB_DEV = core
+RAFT_DEV = $(HOME)/.chaincore/raft
 
-null:
-	@echo Please specify a make target.
+default: run
+
+## run a development version of Chain Core at http://localhost:1999
+run:
+	cored
+
+## reset the development database
+resetdb:
+	-dropdb $(DB_DEV)
+	createdb $(DB_DEV)
+	rm -rf $(RAFT_DEV)
+
+## delete the development database, re-configure core, and run server
+rerun: build-dev corectl resetdb
+	sh -c "corectl wait; corectl config-generator" &
+	cored
+
+## run development dashboard at http://localhost:3000
+dashserver:
```

Comment:
Would it hurt your head if we compromised and did a portmanteau, e.g. `dashserve` and `docserve`?